### ISSUE TITLE
batch update entire column

### ISSUE THREAD

**Issue by zhugw** [mito-ds/monorepo#422]
After filtering , want to batch update one entire column,  e.g.
```
df.loc[df["C"]!="foo3", "C"] = "foo"
```
now mito seems does not support this update operate

**Comment by naterush** [comment]
@zhugw thanks for posting this. Can you help me understanding a little bit better what you want to do here?

It's pretty much: for every row in "C" that does not equal "foo3", update it it to "foo", correct?

If so, is this useful because you're effectively trying to do a find and replace within the column?

**Comment by zhugw** [comment]
Yes, after filtering out the data I want , then want to batch update the entire column, e.g.  there are 100 rows, after filtering column 1 (e.g. c1 = 'foo'), only has 10 rows, then batch update column 2 (c2 = 'bar'), How to implement in Mito?

**Comment by naterush** [comment]
Currently, this is not possible (although something like this is on the update). To clarify, do you then filter to a different value like `different foo` and then update those as well? E.g. are you repeating this process multiple times for the same column?

**Comment by zhugw** [comment]
Very possible. This is routing data correction work. Once some problem is found, it would be modified in batches in excel
